bluesky/run_engine.py

```diff
@@ -508,7 +508,13 @@ def __call__(self, plan, subs=None, *, raise_if_interrupted=False,
         if not self.state.is_idle:
             raise RuntimeError("The RunEngine is in a %s state" % self.state)
 
-        futs = [f for sup in self.suspenders for f in sup.get_futures()]
+        futs = []
+        for sup in self.suspenders:
+            f_lst, msg = sup.get_futures()
+            if f_lst:
+                futs.extend(f_lst)
+                print(msg)
```

Comment:
Good catch - we would like to see the message when it trips mid-scan. @tacaswell just didn't get around to implementing that when this PR went up, if I recall correctly.